less/quicksilver/jakarta/template.less

```diff
@@ -91,6 +91,9 @@ main {
   }
 }
 
+.featured-footer {
+  margin-top: 0px;
```

Comment:
It seems to me that this should be remove from less/_components/quicksilver/featured-footer.less